src/Resource/Repository/RepositoryFactory.js

```diff
@@ -0,0 +1,26 @@
+
+export default function createRepository(
+  manager, repositoryDefinition, connector, resourceType, identifier,
+) {
+  const repository = Object.create(repositoryDefinition, {
+    connector: {
+      writable: false,
+      enumerable: false,
+      value: connector,
+    },
+    resourceType: {
+      writable: false,
+      enumerable: false,
+      value: resourceType,
+    },
+    identifier: {
+      writable: false,
+      enumerable: false,
+      value: identifier,
+    },
+  });
+
+  manager.addRepository(repository);
```

Comment:
Ik denk dat het handig is om een check te doen of de repository al bestaat. 
Nu is het namelijk mogelijk dat code een bestaande repository gebruikt die later overschreven wordt.
Dit hoeft niet erg te zijn, maar volgens mij is het niet de bedoeling.